Source/Atomic/Graphics/AnimatedModel.cpp

```diff
@@ -1044,7 +1044,7 @@ void AnimatedModel::OnWorldBoundingBoxUpdate()
     if (isMaster_)
     {
         // Note: do not update bone bounding box here, instead do it in either of the threaded updates
-        worldBoundingBox_ = boneBoundingBox_.Transformed(node_->GetWorldTransform());
+        worldBoundingBox_ = boundingBox_.Transformed(node_->GetWorldTransform());
     }

```

Comment:
At first any object, excluding prefabs, that had an AnimatedModel component
attached to it could'nt be focused on by pressing the F-key. The issue was
getting the BoundingBox to be drawn again as the F-Key focus and BoundingBox
seems to be related to each other.

Each model's BoundingBox gets created based upon the Center point of the model.
Using the BoneBoundingBox no Center point value could be found. I figured it's 
because the bones don't get created in the scene. So the BoundingBox was never 
drawn.

Using the BoundingBox of the model instead of the BoundingBox of the Bones
a Center point value could be found (probably the center of the mesh?) and the box was drawn and updating properly again.